### PR TITLE
Support option with breaklines, fix multiselect crashing the pages, fix styling

### PR DESCRIPTION
**Pull request by fmontes**
…

### CODE REVIEW

File: projects/dotcms-field-elements/src/components/dot-form/dot-form-fields.tsx [hunk 1]
```diff
@@ -59,7 +59,7 @@ export const DotFormFields = {
             options={field.values}
             required={field.required}
             required-message={field.requiredMessage}
-            size={+field.size}
+            size={field.size}
```
Comment (fmontes) [inline projects/dotcms-field-elements/src/components/dot-form/dot-form-fields.tsx]: This was crashing the page, not sure why.

File: projects/dotcms-field-elements/src/components/dot-multi-select/dot-multi-select.tsx [hunk 2]
```diff
@@ -48,7 +48,7 @@ export class DotMultiSelectComponent {
     @Prop() requiredMessage = `This field is required`;
 
     /** (optional) Size number of the multi-select dropdown (default=3) */
-    @Prop({ reflectToAttr: true }) size = 3;
+    @Prop({ reflectToAttr: true }) size = '3';
```
Comment (fmontes) [inline projects/dotcms-field-elements/src/components/dot-multi-select/dot-multi-select.tsx]: String as default

File: projects/dotcms-field-elements/src/utils/utils.tsx [hunk 3]
```diff
@@ -31,6 +31,8 @@ export function getClassNames(
  * @returns {DotOption[]}
  */
 export function getDotOptionsFromFieldValue(rawString: string): DotOption[] {
+    rawString = rawString.replace(/(?:\\[rn]|[\r\n]+)+/g, ',');
```
Comment (fmontes) [inline projects/dotcms-field-elements/src/utils/utils.tsx]: From the backend the value for the options is coming like: `'Canada|CA\r\nMexico|MX\r\nUSA|US'` so this is for supporting that, I update unit testing.

File: projects/dotcms-field-elements/src/components/dot-form/dot-form-fields.tsx (hunk 1 above)
Comment (alfredo-dotcms) [reply projects/dotcms-field-elements/src/components/dot-form/dot-form-fields.tsx]: did some tests an no idea why I set this, anyway this works!

**Pull request merged**